include/zephyr/mgmt/mcumgr/smp/smp_client.h

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef H_SMP_CLIENT_
+#define H_SMP_CLIENT_
+
+#include <zephyr/net/buf.h>
+#include <mgmt/mcumgr/transport/smp_internal.h>
+
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief SMP client send response callback function
+ *
+ * @param nb net_buf for response
+ *
+ * @param user_data same user data returned than for request
```

Comment:
If you mean the same data, then:
```suggestion
 * @param user_data same user data that was provided as part of the request
```